cea/technologies/chiller_vapor_compression.py

```diff
@@ -74,7 +72,8 @@ def calc_COP(T_cw_in_K, T_chw_re_K, q_chw_load_Wh):
 
 
 def calc_COP_with_carnot_efficiency(peak_cooling_load, q_chw_load_Wh, T_chw_sup_K, T_cw_in_K, VCC_chiller):
-    PLF = calc_averaged_PLF(peak_cooling_load, q_chw_load_Wh, T_chw_sup_K, T_cw_in_K, VCC_chiller)  # calculates the weighted average Part load factor across all chillers based on load distribution
+    PLF = calc_averaged_PLF(peak_cooling_load, q_chw_load_Wh, T_chw_sup_K, T_cw_in_K,
+                            VCC_chiller)  # calculates the weighted average Part load factor across all chillers based on load distribution
```

Comment:
Should this be VC_chiller for consistency then?

Reply:
I changed all occurrences of VCC_chiller to VC_chiller in the last commit.